src/main/java/frc/robot/commands/arm/MoveArm.java

```diff
@@ -46,20 +43,27 @@ public MoveArm(Arm arm, Supplier<ArmPosition> positionSupplier) {
     @Override
     public void initialize() {
         ArmPosition position = armPositionSupplier.get();
-        this.elevatorPosition =
-            MathUtil.clamp(position.getElevatorPosition(), 0, Constants.Elevator.MAX_ENCODER);
         this.wristAngle = position.getWristAngle();
         this.armAngle = position.getArmAngle();
-        arm.enablePID();
         arm.setArmGoal(armAngle);
-        arm.setWristOffset(wristAngle);
-        // arm.setElevatorGoal(elevatorPosition);
+        arm.setWristGoal(wristAngle);
+        // arm.setWristGoal(-25);
+        this.armExtended = position.getArmExtended();
+
+        if (!armExtended) {
+            arm.retractArm();
+        }
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        if (armExtended) {
+            arm.extendArm();
+        }
     }
 
     @Override
     public boolean isFinished() {
-        return arm.checkArmInPosition();
-        // && arm.getWristAligned();
-        // arm.checkElevatorAligned();
+        return false; // arm.checkArmInPosition();
```

Comment:
This can't stay like this for autos. We need something to know that the arm is in position or we'll have to use time only which is not great.

Reply:
That method was removed during arm rewrite. Just pushed something that should fix this.